build.gradle

```diff
@@ -20,9 +20,10 @@ project.ext {
     isCiServer = System.getenv().containsKey("CI")
 }
 
-// Spring Boot 3.2.3 pulls in opentelemetry-bom 1.31.0.
+// Spring Boot 3.4.0 pulls in opentelemetry-bom 1.43.0.
 // We need >= 1.32.0 so that our HttpServerMetrics can use Meter.setExplicitBucketBoundariesAdvice:
-ext['opentelemetry.version'] = '1.42.1'
+ext['opentelemetry.version'] = '1.43.0'
+ext['opentelemetry.instrumentation.version'] = '2.9.0' // 2.9.0 targets opentelemetry 1.43.0
```

Comment:
this is actually a downgrade. We had been on `2.10.0` before this PR. I thought this was more appropriate, since `2.10.0` wants opentelemetry 1.44.1. See https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases.

Reply:
do we need these lines anymore? I think they were there to specify a more advanced version than spring required, is that still true?